### PR TITLE
OPS: Upgrade to React Native 0.59.2

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,18 +83,19 @@
...
     "react-native-snap-carousel": "3.7.5",
     "react-native-sortable-list": "0.0.22",
-    "react-native-svg": "9.1.1",
+    "react-native-svg": "9.3.6",
     "react-native-tcp": "3.3.0",
     "react-native-tooltip": "git+https://github.com/marcosrdz/react-native-tooltip.git",
-    "react-native-vector-icons": "6.2.0",
-    "react-native-webview": "4.1.0",
+    "react-native-vector-icons": "6.4.2",
+    "react-native-webview": "5.5.0",
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply package.json]: Due to changes in RN. latest react-native-webview complies with ReactModule changes.

**Comment by Overtorment** [comment]
not good enough. try harder

**Comment by Overtorment** [comment]
and test lapp browser integration for both ios and android. otherwise PR is good to merge

File: ios/fastlane/metadata/en-US/release_notes.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-v3.9.0
+v3.9.2
```
Comment (Overtorment) [inline ios/fastlane/metadata/en-US/release_notes.txt]: nein

File: App.test.js [hunk 3]
```diff
@@ -231,12 +231,12 @@ it('Wallet can fetch UTXO', async () => {
 it('Wallet can fetch balance', async () => {
   jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
   let w = new LegacyWallet();
-  w._address = '115fUy41sZkAG14CmdP1VbEKcNRZJWkUWG'; // hack internals
+  w._address = '1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa'; // hack internals
```
Comment (Overtorment) [inline App.test.js]: I fixed it in master, and you reverted this in branch.
you need to up your git skillz bro...

File: screen/lnd/browser.js [hunk 4]
```diff
@@ -243,193 +242,103 @@ export default class Browser extends Component {
...
-              this.webview.injectJavaScript(injectedParadise);
-              alreadyInjected = true;
-              console.log('injected');
-            }
-          }}
-        />
-      );
-    } else if (Platform.OS === 'ios') {
-      return (
-        <WKWebView
```
Comment (Overtorment) [inline screen/lnd/browser.js]: There was a reason why I used WKWebView for ios. Dont really remember tho...
Needs good testing on ios. 
I tested on andoid emu - works (good news)

**Comment by Overtorment** [comment]
git history messed up again. redo the PR, without touching unnecessary files like `ios/fastlane/metadata/en-US/release_notes.txt` and not reverting my changes in master.
don't forget to test on ios and android

**Pull request closed without merging**